### DISCUSSION TITLE
TypeError: bokeh.models.tools.Toolbar()

### DISCUSSION THREAD

**Discussion by johnjump457** [kernc/backtesting.py#1050] [Q&A] [answered]
Hi,
I try to plot backtesting result on colab with Python version 3.10.12, but show error bellow :
TypeError: bokeh.models.tools.Toolbar() got multiple values for keyword argument 'logo'

**Comment by kernc** [comment, marked as answer]
See https://github.com/kernc/backtesting.py/issues/1051

**Reply by johnjump457** [reply]
thank you

**Reply by bao292005** [reply]
I am facing the same issue, how did you fix this?

**Reply by eervin123** [reply]
See @kernc 's comment above. See issue [#1051](https://github.com/kernc/backtesting.py/issues/1051)

Install 

bokeh==3.1.1
